src/main/groovy/de/dkfz/eilslabs/tools/constants/Constants.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2016 eilslabs.
+ *
+ * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
```

Comment:
This file must stay within the Roddy core project! It is full of application specific constants.